tests/models/store.py

```diff
@@ -153,15 +152,14 @@ def test_update_ts_plurals(store_po, test_fs):
 
 
 @pytest.mark.django_db
-def test_update_with_non_ascii(en_tutorial_po, test_fs):
-    # Parse store
-    en_tutorial_po.updater.update_from_disk()
-    tp = en_tutorial_po.translation_project
-    with test_fs.open(['data', 'po', tp.real_path,
+def test_update_with_non_ascii(store0, test_fs):
+    store0.state = PARSED
+    orig_units = store0.units.count()
+    with test_fs.open(['data', 'po', 'tutorial', 'en',
                        'tutorial_non_ascii.po']) as f:
         store = getclass(f)(f.read())
-    en_tutorial_po.update(store)
-    assert en_tutorial_po.units[0].target == "Hèllö, wôrld"
+    store0.update(store)
+    assert store0.units[orig_units].target == "Hèllö, wôrld"
```

Comment:
I realise it isn't this PR.  But the non-ascii should take things into the unicode range.  I fear that all these are in the first 127 chars so they're still technically single byte ascii, so not doing what is hoped here.